sys/benchmark/Kconfig

```diff
@@ -7,5 +7,5 @@
 
 config MODULE_BENCHMARK
     bool "Simple benchmarks support"
-    depends on MODULE_XTIMER
+    depends on MODULE_ZTIMER_USEC
```

Comment:
Shouldn't this be `select MODULE_ZTIMER_USEC`. 
[sys/Makefile.dep](https://github.com/RIOT-OS/RIOT/blob/master/sys/Makefile.include) pulls in `ztimer_usec` automatically https://github.com/RIOT-OS/RIOT/blob/c84a40abc4511f2bc5b55f0db8e708432e32d161/sys/Makefile.dep#L574-L576 Kconfig does not.

Reply:
@leandrolanzieri @bergzand ping.

I have an application here that uses `benchmark`. With this PR it is now necessary also to use `ztimer_usec` in application's makefile to be compatible with Kconfig even though `ztimer_usec` is pulled in automatically by the `benchmark` module. Was there any reason not to select `ztimer_usec` in Kconfig with `benchmark`?